src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+
+type Props = {
+  todoList: Todo[];
+  showErrorMessage: (message: string, delay?: number) => void;
+};
+
+export const TodoForm: React.FC<Props> = ({ todoList, showErrorMessage }) => {
+  const [todoTitle, setTodoTitle] = useState('');
+
+  const newTodoRef = useRef<HTMLInputElement>(null);
+
+  const handlerSubmitForm = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (todoTitle === '') {
+      showErrorMessage('Title should not be empty');
+
+      return;
+    }
+
+    showErrorMessage('Unable to add a todo');
+
+    setTodoTitle('');
+
+    // const newTodo: Todo = {
+    //   id: Number(new Date()),
+    //   userId: USER_ID,
+    //   title: newTodoTitle,
+    //   completed: false,
+    // };
+
+    // setTodoList(currentList => [newTodo, ...currentList]);
+  };
+
+  useEffect(() => {
+    if (newTodoRef.current) {
+      newTodoRef.current.focus();
+    }
+  }, [todoList]);
+
+  return (
+    <header className="todoapp__header">
+      {todoList.length !== 0 && (
+        <button
+          type="button"
+          className={classNames('todoapp__toggle-all', {
+            active: todoList?.every(todo => todo.completed === true),
```

Comment:
Move this logic from jsx to the helper variable and use it here
```suggestion
            active: todoList?.every(todo => todo.completed),
```